core/src/components/textarea/test/autogrow/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
```

Comment:
Threading:

What are the input legacy tests updated?

Reply:
The placeholder text color changes between the diffs, but unrelated to these changes as far as I am aware.